network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkImpl.java

```diff
@@ -45,6 +44,8 @@ public class NetworkImpl extends AbstractIdentifiableImpl<Network, NetworkAttrib
 
     private final List<NetworkListener> listeners = new ArrayList<>();
 
+    private AbstractReporterContext reporterContext;
+
     public NetworkImpl(NetworkStoreClient storeClient, Resource<NetworkAttributes> resource) {
         super(new NetworkObjectIndex(storeClient), resource);
         index.setNetwork(this);
```

Comment:
```suggestion
        this.reporterContext = new SimpleReporterContext();
        index.setNetwork(this);
```
Should we add a reporter context here to match the implementation in core (https://github.com/powsybl/powsybl-core/blob/main/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java#L128) ?